pkg/controller/permissions/permissions_check.go

```diff
@@ -63,9 +63,8 @@ func CheckPermissions(req router.Request, _ router.Response) error {
 	}
 
 	// Early exit
-	if (app.Status.Staged.AppImage.ID == "" ||
-		app.Status.Staged.AppImage.Digest == app.Status.AppImage.Digest) &&
-		app.Status.Staged.PermissionsObservedGeneration == app.Generation {
+	if app.Status.Staged.AppImage.ID == "" ||
+		app.Status.Staged.AppImage.Digest == app.Status.AppImage.Digest && app.Status.Staged.PermissionsObservedGeneration == app.Generation {
```

Comment:
Super small nitpick.
```suggestion
	if app.Status.Staged.AppImage.ID == "" ||
		app.Status.Staged.AppImage.Digest == app.Status.AppImage.Digest &&
		app.Status.Staged.PermissionsObservedGeneration == app.Generation {
```

Reply:
Eh, not really worth bringing up.